evap/static/ts/src/contact_modal.ts

```diff
@@ -0,0 +1,64 @@
+declare const csrf: typeof import("./csrf-utils");
+declare const bootstrap: typeof import("bootstrap");
+
+class ContactModalLogic {
+    private readonly modal: bootstrap.Modal;
+    private readonly successMessageModal: bootstrap.Modal;
+    private readonly actionButtonElement: HTMLButtonElement;
+    private readonly messageTextElement: HTMLInputElement;
+    private readonly modal_id: string;
```

Comment:
We want to stick to one naming style, the default in javascript is camel case, so let's change it to
```suggestion
    private readonly messageTextElement: HTMLInputElement;
    private readonly modalId: string;
```